src/pages/Account/Admin.tsx

```diff
@@ -223,13 +241,17 @@ const EventsComponent: React.FC<VenueDetailsPartProps> = ({ venue }) => {
           Create an Event
         </button>
       </div>
-      <AdminEvent
-        show={showCreateEventModal}
-        onHide={() => {
-          setShowCreateEventModal(false);
-        }}
-        venueId={venue.id}
-      />
+      {showCreateEventModal && (
```

Comment:
does unmounting the Modal instead of using the show prop to show/hide it still show the fade away animation?

Reply:
It does. Unmounting the modal get the form to take the default props into account

Reply:
great if it does. If it doesn't, the way to handle defaultProps would be to unmount the sub component which instantiates the form inside the modal itself based on the `show` prop.